src/Adapter/Product/Combination/CommandHandler/UpdateCombinationPricesHandler.php

```diff
@@ -60,6 +73,31 @@ public function handle(UpdateCombinationPricesCommand $command): void
         $combination = $this->combinationRepository->get($command->getCombinationId());
         $updatableProperties = $this->fillUpdatableProperties($combination, $command);
         $this->combinationRepository->partialUpdate($combination, $updatableProperties, CannotUpdateCombinationException::FAILED_UPDATE_PRICES);
+        if (null !== $command->getWholesalePrice()) {
+            $this->updateDefaultSupplier($command->getCombinationId(), $command->getWholesalePrice());
+        }
+    }
+
+    private function updateDefaultSupplier(CombinationId $combinationId, DecimalNumber $wholesalePrice): void
+    {
+        $productId = $this->combinationRepository->getProductId($combinationId);
+        $defaultSupplierId = $this->productSupplierRepository->getDefaultSupplierId($productId);
+        if (!$defaultSupplierId) {
+            return;
+        }
+
+        $defaultProductSupplierId = $this->productSupplierRepository->getIdByAssociation(new ProductSupplierAssociation(
+            $productId->getValue(),
+            $combinationId->getValue(),
+            $defaultSupplierId->getValue()
+        ));
+        if (!$defaultProductSupplierId) {
+            return;
+        }
+
+        $defaultProductSupplier = $this->productSupplierRepository->get($defaultProductSupplierId);
+        $defaultProductSupplier->product_supplier_price_te = (float) (string) $wholesalePrice;
```

Comment:
That's the legacy name indeed so we won't change it to avoid a breaking change

At least the field explicitly indicates that the value is tax excluded (te = tax excluded), in most of the DB prices don't have a suffix but they are mostly tax excluded which must be know by convention 🤷‍♂️. So it's not that bad 😅